source/user-manual/capabilities/file-integrity/advanced-settings.rst

```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
          # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
          # rc-service auditd restart
 
+.. note::
```

Comment:
Why a note? Why not just inserting all this in the enabling who-data monitoring steps that follow below?

Reply:
Paragraph modified with a new layout outside the note!